packages/calcite-components/src/components/time-picker/time-picker.tsx

```diff
@@ -76,6 +76,14 @@ export class TimePicker
   //
   //--------------------------------------------------------------------------
 
+  /** Formats the displayed time value in either 12 or 24 hour format.  Defaults to the `lang`'s preferred setting. */
```

Comment:
Same suggestion as above WRT the format reference:

```suggestion
  /** Displays the component's `value` in either 12 or 24 hour format.  Defaults to the `lang`'s preferred setting. */
```